### DISCUSSION TITLE
Check existing and new saved data like in Firebase

### DISCUSSION THREAD

**Discussion by LebedevIV** [appy-one/acebase-server#76] [Q&A] [answered]
I carefully read the manual, but did not understand whether it is possible to check existing and new saved data like in Firebase: for example, data.exists() in relation to existing data or newData.hasChildren(['TypeInfo', 'URI']) to new ? Or do I need to make some custom classes of my own?

**Comment by appy-one** [comment, marked as answer]
I have been working on this for #70, might be able to commit code for this Monday

**Reply by appy-one** [reply]
I have finished implementing this, has been published with v1.17.0. Kindly note that the implementation is slightly different from Firebase's, not less powerful though! Read the docs for more info.

Do you like AceBase? 👇🏼 

[![Sponsor AceBase](https://user-images.githubusercontent.com/26569719/168233053-8e56b243-4140-40ab-9a30-4cb3cc149bfe.svg)](https://github.com/sponsors/appy-one) [![Spread the word](https://user-images.githubusercontent.com/26569719/169265089-3d593555-e1ad-4390-986b-877ac2c38a47.svg)](https://twitter.com/intent/tweet?button=&url=https://github.com/appy-one/acebase&text=I'm+using+@AcebaseRealtime+in+my+project+to+make+my+life+easier!&button=)  [![contribute](https://user-images.githubusercontent.com/26569719/169265318-30c4c6a5-7c89-46a0-a7a2-ef433a8192f4.svg)](https://github.com/appy-one/acebase#contributing)

**Reply by LebedevIV** [reply]
My system is still implemented on Firebase and I could not make a timely transfer to Icebase, although this is in the plans. Family problems interfere, very little time. But I understand that now Icebase fully meets my expectations.
